code/game/machinery/Sleeper.dm

```diff
@@ -1,3 +1,8 @@
+#define SLEEPER_TEND		"Treat Injuries"
+#define SLEEPER_ORGANS		"Repair Organs"
+#define SLEEPER_CHEMPURGE	"Purge Toxins"
+#define SLEEPER_HEAL_RATE 2
```

Comment:
You should probably reduce this back to 1, 2 is quite a lot

Reply:
2 is about the same as reagent healing, considering the patient can't just walk out of the sleeper after a second I don't think it needs to be much lower and the initial values were contested for being extremely slow

Reply:
As I said above, I still think having the heal rate be lower (1) with the ability to be bumped up to typical chem healing rates (2 fir aiuri, 3 for libital) by filling it with those chems in a buffer would be a good passive / active system compromise.

Reply:
I'd rather if people want chems they have to get them from the chemist, if they need automatic chemical application they get it from cryo

Reply:
yeah from live testing 2 is still slow